nri-log-explorer/src/Main.hs

```diff
@@ -907,77 +913,81 @@ viewSpanBreakdownWithMatch matching span =
 
 viewSpanDetails :: Span -> Brick.Widget Name
 viewSpanDetails Span {original} =
-  Brick.vBox
-    [ viewDetail "name" (Platform.name original),
-      case Platform.summary original of
-        Nothing -> Brick.emptyWidget
-        Just summary -> viewDetail "summary" summary,
-      viewDetail
-        "duration"
-        ( ( Platform.finished original - Platform.started original
-              |> Platform.inMicroseconds
-              |> Prelude.fromIntegral
-              |> (\n -> n `Prelude.div` 1000)
-              |> Text.fromInt
-          )
-            ++ " ms"
-        ),
-      case Platform.frame original of
-        Nothing -> Brick.emptyWidget
-        Just (_, srcLoc) ->
-          viewDetail
-            "source"
-            ( Data.Text.pack (Stack.srcLocFile srcLoc)
-                ++ ":"
-                ++ Text.fromInt (Prelude.fromIntegral (Stack.srcLocStartLine srcLoc))
-            ),
-      case Platform.succeeded original of
-        Platform.Succeeded -> viewDetail "result" "succeeded"
-        Platform.Failed -> viewDetail "result" "failed"
-        Platform.FailedWith exception ->
+  Brick.viewport
```

Comment:
I find it interesting that you added this inside `viewSpanDetails` where as I would have added it outside. I don't think it matters, but it's interesting how preferences are different.
I would have added it outside, because I feel like it's not part of rendering span details.

Reply:
Also because you can write
```hs
viewSpanDetails span
  |> Brick.viewport SpanDetail Vertical
```
without having so much nesting inside here.

Reply:
> I find it interesting that you added this inside viewSpanDetails where as I would have added it outside. I don't think it matters, but it's interesting how preferences are different.

FWIW I also think that having it outside is better (reasoning: that feels more like "how it fits into the parent's layout"). I suppose that at the time I was so new to the code I didn't really understand what I was doing well enough to make that type of decision :-P